src/QMCWaveFunctions/BsplineFactory/HybridRepSetReader.h

```diff
@@ -561,51 +558,6 @@ class HybridRepSetReader : public SplineSetReader<SA>
           }
         }
       }
-
-#ifdef PRINT_RADIAL
-      char fname[64];
```

Comment:
Is @ye-luo okay with this debug code being deleted?

Reply:
Fine with the deletion. I can restore these lines if I really need them.

Reply:
I tried defining PRINT_RADIAL and the debug code is broken (`center_idx` isn't defined)